tests/Feature/TermsTest.php

```diff
@@ -44,4 +45,30 @@ function a_term_that_only_exists_in_english_shows_it_in_english_on_other_locales
             ->assertSee($term->name)
             ->assertSee($term->description);
     }
+
+    /** @test */
+    function a_term_may_be_related_to_one_or_more_resources()
```

Comment:
I saw your note in the description about this. I don't quite understand what you're saying--is it that we need to test that someone doesn't remove this relationship method later down the road? That doesn't seem to be worth testing for me.